ras_backstage/controllers/collection_instrument_controller.py

```diff
@@ -0,0 +1,27 @@
+import json
+import logging
+
+from structlog import wrap_logger
+
+from ras_backstage import app
+from ras_backstage.common.requests_handler import request_handler
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def upload_collection_instrument(survey_id, collection_exercise_id, file):
+    logger.debug('Uploading collection instrument', collection_exercise_id=collection_exercise_id)
```

Comment:
Would it be worth including the `survey_id` in the log message too since we have it here? 
The more info we can be logging out the better probably